etc/docker/test/matrix.yml

```diff
@@ -16,14 +16,7 @@ dists:
       python:
         - "3.9"
 python:
-  - id: "3.6"
-    allow:
-      suites:
-        - syntax
-        - docs
-        - client_syntax
-        - remote_dbs
-        - multi_vo
+  - "3.6"
```

Comment:
```
- id: 3.6
```

Reply:
doesn't matter, if the entry is not a dict, it will be interpreted as the id: https://github.com/rucio/rucio/blob/2dfb00ef4e032867170aa5b02abb4272e594a9dd/tools/test/matrix_parser.py#L33-L34

I just reverted https://github.com/rucio/rucio/commit/9b5d43ca0a288bdbd3dd97573d5a082464af4081#diff-3273b199b4da4a866dddefe08c4fc04f3d69243f9472b929669ac5cc0364f608R16-R23